plugin.video.catchuptvandmore/addon.xml

```diff
@@ -21,7 +21,7 @@
   <extension point="xbmc.addon.metadata">
     <!-- Set reuselanguageinvoker to false while developing the addon, making it only true for the production version -->
     <!-- See https://kodi.wiki/view/Addon.xml#.3Creuselanguageinvoker.3E -->
-    <reuselanguageinvoker>true</reuselanguageinvoker>
+    <!-- <reuselanguageinvoker>true</reuselanguageinvoker> -->
```

Comment:
intentional @SylvainCecchetto ?

Reply:
Yes, because of this https://github.com/ruuk/script.module.youtube.dl/pull/56